doc/nrf/introduction.rst

```diff
@@ -55,10 +55,10 @@ The version numbers are incremented based on the following criteria:
 
 * The ``MAJOR`` version number is increased seldom, whenever a release is deemed to be introducing a large number of substantial changes across the board.
 * The ``MINOR`` version number is increased every time a major release is cut.
-  Major releases are the default types of an |NCS| release.
+  Minor releases are the default types of an |NCS| release.
   They introduce new functionality and may break APIs.
 * The ``PATCH`` version number is increased whenever a minor or bugfix release is cut.
```

Comment:
Also, @carlescufi , @shantha-14 , we do not define what is a bugfix release anywhere on this page...

Reply:
I think we should define bug fix release. Also state that a bug fix tag is done from release branches.